### PR TITLE
[mod_conference] Fix possible NULL argument.

### PR DESCRIPTION
**Pull request by iguanae**
On branch issue_210
Changes to be committed:
       modified:   src/mod/applications/mod_conference/conference_event.c

### CODE REVIEW

File: src/mod/applications/mod_conference/conference_event.c [hunk 1]
```diff
@@ -108,7 +108,8 @@ void conference_event_mod_channel_handler(const char *event_channel, cJSON *json
 		}
 	}
 
-	switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_DEBUG, "conf %s CMD %s [%s] %s\n", conference_name, key, action, cid);
+//	switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_DEBUG, "conf %s CMD %s [%s] %s\n", conference_name, key, action, cid);
```
Comment (mjerris) [inline src/mod/applications/mod_conference/conference_event.c]: please remove the commented out line from patch.

File: src/mod/applications/mod_conference/conference_event.c [hunk 2]
```diff
@@ -339,7 +340,7 @@ void conference_event_mod_channel_handler(const char *event_channel, cJSON *json
 				exec = switch_mprintf("%s cam %d %d pan=%s:%d%s", conference_name, canvas_id, layer_id, xy, metric, i);
 			}
 		}
-
```
Comment (mjerris) [inline src/mod/applications/mod_conference/conference_event.c]: please revert all whitespace changes from the patch

File: src/mod/applications/mod_conference/conference_event.c [hunk 3]
```diff
@@ -352,8 +353,8 @@ void conference_event_mod_channel_handler(const char *event_channel, cJSON *json
 		if ((v = cJSON_GetObjectItem(data, "canvasID"))) {
 			canvas_id = v->valueint;
 		}
-
```
Comment (mjerris) [inline src/mod/applications/mod_conference/conference_event.c]: please revert all whitespace changes from the patch

File: src/mod/applications/mod_conference/conference_event.c [hunk 4]
```diff
@@ -377,7 +378,7 @@ void conference_event_mod_channel_handler(const char *event_channel, cJSON *json
 		if (canvas_id > -1 && layer_id > -1 && x > -1 && y > -1 && w > -1 && h > -1) {
 			exec = switch_mprintf("%s cam %d %d zoom=%d:%d:%d:%d snap_factor=1 zoom_factor=1", conference_name, canvas_id, layer_id, x, y, w, h);
 		}
-
```
Comment (mjerris) [inline src/mod/applications/mod_conference/conference_event.c]: please revert all whitespace changes from the patch

File: src/mod/applications/mod_conference/conference_event.c [hunk 5]
```diff
@@ -586,17 +587,17 @@ void conference_event_adv_layout(conference_obj_t *conference, mcu_canvas_t *can
 
 	cJSON_AddItemToObject(msg, "eventChannel", cJSON_CreateString(conference->info_event_channel));
 	cJSON_AddItemToObject(data, "contentType", cJSON_CreateString("layout-info"));
-
```
Comment (mjerris) [inline src/mod/applications/mod_conference/conference_event.c]: please revert all whitespace changes from the patch

File: src/mod/applications/mod_conference/conference_event.c [hunk 6]
```diff
@@ -586,17 +587,17 @@ void conference_event_adv_layout(conference_obj_t *conference, mcu_canvas_t *can
 
 	cJSON_AddItemToObject(msg, "eventChannel", cJSON_CreateString(conference->info_event_channel));
 	cJSON_AddItemToObject(data, "contentType", cJSON_CreateString("layout-info"));
-	
+
 	switch_mutex_lock(canvas->mutex);
-
```
Comment (mjerris) [inline src/mod/applications/mod_conference/conference_event.c]: please revert all whitespace changes from the patch

File: src/mod/applications/mod_conference/conference_event.c [hunk 7]
```diff
@@ -586,17 +587,17 @@ void conference_event_adv_layout(conference_obj_t *conference, mcu_canvas_t *can
...
-	
+
 	if ((obj = get_canvas_info(canvas))) {
 		cJSON *array = cJSON_CreateArray();
 
 		for (i = 0; i < vlayout->layers; i++) {
 			cJSON *layout = cJSON_CreateObject();
 			int scale = vlayout->images[i].scale;
 			int hscale = vlayout->images[i].hscale ? vlayout->images[i].hscale : scale;
-
```
Comment (mjerris) [inline src/mod/applications/mod_conference/conference_event.c]: please revert all whitespace changes from the patch

File: src/mod/applications/mod_conference/conference_event.c [hunk 8]
```diff
@@ -614,7 +615,7 @@ void conference_event_adv_layout(conference_obj_t *conference, mcu_canvas_t *can
 			cJSON_AddItemToObject(layout, "audioPOS", cJSON_CreateString(vlayout->images[i].audio_position));
 			cJSON_AddItemToArray(array, layout);
 		}
-
```
Comment (mjerris) [inline src/mod/applications/mod_conference/conference_event.c]: please revert all whitespace changes from the patch

File: src/mod/applications/mod_conference/conference_event.c (hunk 1 above)
Comment (iguanae) [reply src/mod/applications/mod_conference/conference_event.c]: 52d94ae removes the commented out line and restore all whitespaces. 

Thanks.

**Pull request merged**